[2025-05-16T22:09:40.888+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-16T22:09:40.911+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_stock_news_and_sentiment_dag.fetch_news_and_sentiments manual__2025-05-16T22:06:28.423629+00:00 [queued]>
[2025-05-16T22:09:40.922+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_stock_news_and_sentiment_dag.fetch_news_and_sentiments manual__2025-05-16T22:06:28.423629+00:00 [queued]>
[2025-05-16T22:09:40.923+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-16T22:09:40.940+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_news_and_sentiments> on 2025-05-16 22:06:28.423629+00:00
[2025-05-16T22:09:40.947+0000] {standard_task_runner.py:72} INFO - Started process 3462 to run task
[2025-05-16T22:09:40.952+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_stock_news_and_sentiment_dag', 'fetch_news_and_sentiments', 'manual__2025-05-16T22:06:28.423629+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/daily_news_sentiment_dag.py', '--cfg-path', '/tmp/tmpu11wxpet']
[2025-05-16T22:09:40.954+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask fetch_news_and_sentiments
[2025-05-16T22:09:41.018+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_stock_news_and_sentiment_dag.fetch_news_and_sentiments manual__2025-05-16T22:06:28.423629+00:00 [running]> on host 9f5193fcdf75
[2025-05-16T22:09:41.110+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_stock_news_and_sentiment_dag' AIRFLOW_CTX_TASK_ID='fetch_news_and_sentiments' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T22:06:28.423629+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-16T22:06:28.423629+00:00'
[2025-05-16T22:09:41.112+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-16T22:09:41.113+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-16T22:09:41.113+0000] {logging_mixin.py:190} INFO - Current task name:fetch_news_and_sentiments state:running start_date:2025-05-16 22:09:40.912859+00:00
[2025-05-16T22:09:41.114+0000] {logging_mixin.py:190} INFO - Dag name:daily_stock_news_and_sentiment_dag and current dag run status:running
[2025-05-16T22:09:41.114+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-16T22:09:42.873+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/daily_news_sentiment_dag.py", line 134, in fetch_news_and_sentiments
    tokenizer = AutoTokenizer.from_pretrained("yiyanghkust/finbert-tone")
NameError: name 'AutoTokenizer' is not defined
[2025-05-16T22:09:42.893+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-05-16T22:09:42.894+0000] {logging_mixin.py:190} INFO - Task start:2025-05-16 22:09:40.912859+00:00 end:2025-05-16 22:09:42.893224+00:00 duration:1.980365
[2025-05-16T22:09:42.895+0000] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): fetch_news_and_sentiments> dag:<DAG: daily_stock_news_and_sentiment_dag> dagrun:<DagRun daily_stock_news_and_sentiment_dag @ 2025-05-16 22:06:28.423629+00:00: manual__2025-05-16T22:06:28.423629+00:00, state:running, queued_at: 2025-05-16 22:06:28.525915+00:00. externally triggered: True>
[2025-05-16T22:09:42.896+0000] {logging_mixin.py:190} INFO - Failure caused by name 'AutoTokenizer' is not defined
[2025-05-16T22:09:42.897+0000] {taskinstance.py:1226} INFO - Marking task as UP_FOR_RETRY. dag_id=daily_stock_news_and_sentiment_dag, task_id=fetch_news_and_sentiments, run_id=manual__2025-05-16T22:06:28.423629+00:00, execution_date=20250516T220628, start_date=20250516T220940, end_date=20250516T220942
[2025-05-16T22:09:42.964+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-16T22:09:42.965+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 25 for task fetch_news_and_sentiments (name 'AutoTokenizer' is not defined; 3462)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/daily_news_sentiment_dag.py", line 134, in fetch_news_and_sentiments
    tokenizer = AutoTokenizer.from_pretrained("yiyanghkust/finbert-tone")
NameError: name 'AutoTokenizer' is not defined
[2025-05-16T22:09:43.023+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-05-16T22:09:43.076+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-16T22:09:43.078+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
